geode-core/src/main/java/org/apache/geode/cache/Region.java

```diff
@@ -1362,49 +1362,77 @@ Object selectValue(String queryPredicate) throws FunctionDomainException, TypeMi
    * <code>Scope.LOCAL</code>.
    *
    * If an exception is thrown due to this call, it can imply that there may have been a partial
-   * update performed on the region. Use putAll from within a transaction to to obtain an atomic
-   * update.
+   * update performed on the region.
    * <p>
    *
    * @see java.util.Map#putAll(Map map)
-   * @param map the key/value pairs to put in this region.
+   * @param map the key/value pairs to put in this region
+   * @throws ServerOperationException if called from a client, and the server throws an exception
+   *         such as CacheWriterException, PartitionedRegionStorageException or LowMemoryException.
+   *         These server exceptions become the ServerOperationException cause
+   * @throws ServerConnectivityException if called from a client, and the server throws
+   *         CancelException. CancelException will be the ServerConnectivityException cause
    * @throws NullPointerException if key is null or if value is null (use invalidate instead), or if
    *         the key or value do not meet serializability requirements
```

Comment:
Please slightly reword NullPointerException (in BOTH putAll signatures):
if the key is null, if the value is null (use invalidate instead), or if the key or value do not meet serialization requirements

Reply:
fixed.